src/maintainer/knowledge_base.rst

```diff
@@ -1518,3 +1518,61 @@ The tl;dr here is that conda sorts as follows:
 So make sure that you **tag** your package in such a way that the package name
 that conda-build spits out will sort the package uploaded with an ``rc`` label
 higher than the package uploaded with the ``dev`` label.
+
+Perl packages
+=============
+
+Perl has three standard install locations: **core**, **vendor**, and **site**. Here, **core** is used only for the perl itself, **vendor** is used for installing any other conda-forge packages, and **site** is used for the user to install things locally from sources other than conda-forge.
+
+The most commonly used build system for `perl packages <https://github.com/conda-forge/perl-file-which-feedstock>`__ is ``ExtUtils::MakeMaker``. For the archetypical packaging of packages of this form, you can have a look at `perl-file-which recipe <https://github.com/conda-forge/perl-file-which-feedstock/blob/main/recipe/meta.yaml>`__.
+
+A few things to note in `perl-file-which recipe <https://github.com/conda-forge/perl-file-which-feedstock/blob/main/recipe/meta.yaml>`__ are:
+
+  * The ``name`` is composed of ``perl-`` **+** the lower-cased version of the CPAN name.
+
+      .. code-block::
+
+          package:
+            name: perl-{{ name|lower }}
+            version: {{ version }}
+
+  * The requirements section contain ``make`` for build directive, ``perl`` and ``perl-extutils-makemaker`` for host directive, and ``perl`` for run directive.
```

Comment:
```suggestion
  * The requirements section contain ``make`` for the build directive, ``perl`` and ``perl-extutils-makemaker`` for the host directives, and ``perl`` for the run directive.
```

Reply:
@beckermr Just checking, am I on the right track with these edits?

Reply:
I am a total perl n00b. Maybe someone from @conda-forge/core can help? @mbargull?

Reply:
We may want to ask the bioconda folks.

Reply:
I bumped bioconda core in the shared gitted channel.

Reply:
Leave it as `host directive` singular, as there is only one `host:` section.

But I would drop “directive” here anyway and mention other dependencies:

> The requirements section contains ``make`` for build, ``perl`` and ``perl-extutils-makemaker`` for host, and ``perl`` for run, in addition to any other packages that the particular Perl module being bullt may require.